Added group full detection.

dowzhong opened the pull request:
We still need to make the bot proactively send a message when a group becomes full.

dowzhong committed:
Added group full detection.

We still need to make the bot proactively send a message when a group becomes full.

NorbiPeti commented on src/config.js:
```diff
@@ -12,20 +12,46 @@ config.save = function () {
+            }
+          }else if(config.data[GUILD_ID][ROLE_ID].members.length == config.data[GUILD_ID].games[i][1]){
+            //MAYBE PROACTIVELY SEND A MESSAGE TO THE GUILD IF THE GROUP IS FULL?
+          }else{
```
I think this block only runs if there are 1 more people than the max.

NorbiPeti commented on src/config.js:
```diff
@@ -12,20 +12,46 @@ config.save = function () {
         config.data[GUILD_ID] = {}
     }
     config.data[GUILD_ID][ROLE_ID] = {
+        creator: USER_ID,
```
I'm not sure if we should treat the creator any differently than the other users. Nobody can be added twice and I think the creator should be allowed to leave their room, if they want to cancel it or something.

NorbiPeti committed:
Merge branch 'Dev' into Dev

dowzhong committed:
Fixed up detecting when a group becomes full.

If a group becomes full, it is returned in the addUser promise as 'full'. you can modify the message it sends in bot.js
